_episodes/09-pull-requests.md

```diff
@@ -84,32 +128,66 @@ $ git remote -v
 ~~~
 origin	https://github.com/USERNAME/countries.git (fetch)
 origin	https://github.com/USERNAME/countries.git (push)
-upstream	https://github.com/McMahonLab/countries.git (fetch)
-upstream	https://github.com/McMahonLab/countries.git (push)
+upstream	https://github.com/INSTRUCTOR-GIVEN/countries.git (fetch)
+upstream	https://github.com/INSTRUCTOR-GIVEN/countries.git (push)
 ~~~
 {: .output}
 
-Now that we have this setup done, which is only needed the first time, we can now suggest changes to this repo.
-Each person will add info about a specific country to the repo.
-Since you cloned the repo a template file with info on the United States has been added.
-First we will update our local version of the repo, with this change.
+Now that we have this setup done we will be able to suggest 
+changes to this repo using a pull request.
+Each person will add a new file with info about a new country in it.
+
+The instructor will now add a single file to the repository containing 
+information about the the United States.
+
+Next, we will update our local version of the repo, with this change by `pull`ing down
+the changes from the 'upstream' version's master branch.
```

Comment:
In new version I changed version to remote and fleshed out the description a bit.  Let me know if you think it needs more work (when I push next).

P.S. I really appreciate your noticing when it isn't simplified enough.  Very helpful!